amqp/src/main/scala/akka/stream/alpakka/amqp/scaladsl/AmqpPublishFlow.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.amqp.scaladsl
+
+import akka.Done
+import akka.stream.alpakka.amqp.{AmqpSinkSettings, OutgoingMessage}
+import akka.stream.alpakka.amqp.impl.AmqpPublishFlowStage
+import akka.stream.scaladsl.{Flow, Keep}
+import akka.util.ByteString
+
+import scala.concurrent.Future
+
+object AmqpPublishFlow {
```

Comment:
I think `AmqpFlow` would be enough as publishing can be assumed to be the default.